pkg/metrics/otel.go

```diff
@@ -140,88 +140,109 @@ processors:
           - remediate_duration_seconds
           - resource_conflicts_total
           - internal_errors_total
-          - rendering_count_total
```

Comment:
i dont see some of these below. are they being deliberately removed?

Reply:
These metrics no longer exist in Config Sync, so clearing them up from the allowlist.